src/components/UserManagement/ActivationDatePopup.jsx

```diff
@@ -13,7 +14,7 @@ const ActivationDatePopup = React.memo(props => {
     props.onClose();
   };
   const pauseUser = () => {
-    if (Date.parse(activationDate) > Date.now()) {
+    if (moment().isBefore(moment(activationDate))) {
```

Comment:
So activationDate also information of what timezone the user was when he paused it? Please provide steps to test this PR.

Reply:
<img width="712" alt="image" src="https://user-images.githubusercontent.com/108507783/215306959-b7908b5b-a099-40ed-a612-f8d8b4fb5f8e.png">
When user select the date , (moment().isBefore(moment(activationDate)) will compare the user's local time and the date in local time zone.